File: 0392-is-subsequence/0392-is-subsequence.js
/**
 * @param {string} s
 * @param {string} t
 * @return {boolean}
 */
var isSubsequence = function(s, t) {
    let slow=0;
    for(let i=0;i<t.length;i++){
        if(s[slow]===t[i]){
            slow+=1
        }

    }
    return slow===s.length
    
};

File: 0392-is-subsequence/0392-is-subsequence.py
class Solution(object):
    def isSubsequence(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        if len(s)>len(s):
            return False
        slow=0
        for i in range(len(t)):
            if slow==len(s):
                break
            if s[slow]==t[i]:
                slow+=1
            
        if slow==len(s):
            return True
        else:
            return False


        